exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -123,23 +131,42 @@ class OTLPExporterMixin(
         endpoint: OpenTelemetry Collector receiver endpoint
         credentials: ChannelCredentials object for server authentication
         metadata: Metadata to send when exporting
+        compression: Compression algorithm to be used in channel
     """
 
     def __init__(
         self,
         endpoint: str = "localhost:55680",
         credentials: ChannelCredentials = None,
         metadata: Optional[Tuple[Any]] = None,
+        compression: OTLPCompression = OTLPCompression.NO_COMPRESSION,
```

Comment:
On second thought, it would better to default to `None` here so that users can specify no compression in the code even if the envvar is saying gzip.